core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -319,15 +319,15 @@ private[spark] class AppStatusListener(
     }
 
     val lastStageInfo = event.stageInfos.sortBy(_.stageId).lastOption
-    val lastStageName = lastStageInfo.map(_.name).getOrElse("(Unknown Stage Name)")
+    val jobName = lastStageInfo.map(_.name).getOrElse("(Unknown job without stages)")
```

Comment:
I'm OK with this... I guess the _job_ isn't unknown, but its _name_ is. "Unknown job name" would be more accurate, or a blank string. It doesn't so much matter that it has no stages.

Reply:
OK, I just keep it blank first.